src/shared_components/HelpMe/Content.js

```diff
@@ -148,22 +145,21 @@ const HelpMe = ({ tripId, session, tripParent, isLoadingUser, user }) => {
   const askForQuote = () => {
     setErrors({});
     const data = {
-      user_id: session.username && session._id,
```

Comment:
@fcmatteo we don't need the userId, as it'll be inferred by the accessToken that the user uses to send the request.